### PR TITLE
borgbackup: Update to 1.1.10

### PR DESCRIPTION
**Pull request by gduchene**
#### Description
This version bundles msgpack-python to avoid compatibility issues with
later versions.

Closes: https://trac.macports.org/ticket/58215

###### Type(s)
- [x] update
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.5 18F132
Xcode 10.3 10G8

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by gduchene** [comment]
Looks like the only build that failed was due to a transient error:

```
Installing dependency (48 of 53) 'py36-sphinxcontrib-jsmath' with variants '' (requesting '') ... [FAIL] (fetch)
```

File: sysutils/borgbackup/Portfile [hunk 1]
```diff
@@ -23,11 +23,9 @@ long_description    BorgBackup (short: Borg) is a deduplicating backup \
...
-checksums           rmd160  b40a6bf068d557bd7359ca77da53417fe403092e \
-                    sha256  7d0ff84e64c4be35c43ae2c047bb521a94f15b278c2fe63b43950c4836b42575 \
-                    size    3468259
-
-patchfiles          patch-msgpack.diff
+checksums           rmd160  4d02f9363b585a703c90854f22f8260cc596c84a \
+                    sha256  efb41416d24ff1d13c7952c7f4eaf41ef6fc5e1000354217db55cd62c905e7de \
+                    size    3610011
 
 python.default_version  36
```
Comment (mf2k) [inline sysutils/borgbackup/Portfile]: Can you use python 37 here instead?

Comment (gduchene) [reply sysutils/borgbackup/Portfile]: Sure can!

**Comment by mf2k** [comment]
Please mention the change to python37 in the commit message.

**Comment by gduchene** [comment]
Sure, done.

**Pull request merged**